src/test/java/core/basesyntax/UserTest.java

```diff
@@ -0,0 +1,93 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class UserTest {
+    private RegistrationServiceImpl registration;
+    private User user;
+    private StorageDao storageDaoImpl;
+
+    @BeforeEach
+    void setUp() {
+        registration = new RegistrationServiceImpl();
+        storageDaoImpl = new StorageDaoImpl();
+        user = new User();
+        user.setLogin("Ivan Zhuravlev");
+        user.setAge(18);
+        user.setPassword("my password 123");
+    }
+
+    @Test
+    void register_Ok() {
+        User actual = registration.register(user);
+        assertEquals(user, actual);
+    }
+
+    @Test
+    void addedToStorage_Ok() {
+        registration.register(user);
+        assertEquals(user, storageDaoImpl.get("Ivan Zhuravlev"));
+    }
+
+    @Test
+    void objectUserIsNull_NotOk() {
+        assertThrows(RuntimeException.class, () -> {
+            registration.register(null);
+        });
```

Comment:
```suggestion
        assertThrows("you own exception".class, () -> {
            registration.register(null);
        },"Expected " + "you own exception".class.getName()
                + " to be thrown for the null user, but it wasn't");
        ```
we should write like this. Do this with all same test